scripts/update-global-changelog.js

```diff
@@ -0,0 +1,185 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+
+*/
+
+import fs from 'fs';
+import { execSync } from 'child_process';
+import { fileURLToPath } from 'url';
+import path from 'path';
+import { version as newVersion } from '@spectrum-web-components/base/src/version.js';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const repoUrl = 'https://github.com/adobe/spectrum-web-components';
+
+async function createGlobalChangelog() {
```

Comment:
Since this is a new script, can we include the JSDoc notes on all the new functions just to keep our stuff well documented. Might be good to add a note about when this script needs to be run.